assets/js/base/components/price-slider/utils.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { NumberFormatValues } from 'react-number-format';
+
+export const withMaxValueLimit = ( {
+	maxConstraint,
+	minorUnit,
+}: {
+	maxConstraint: number;
+	minorUnit: number;
+} ) => ( { floatValue }: NumberFormatValues ): boolean => {
+	const maxPrice = maxConstraint / 10 ** minorUnit;
```

Comment:
We probably don't want to use this logic here given that we had issues with it, preferably we just deal with raw versions `1000` and let the final UI element handle formatting.

Reply:
> let the final UI element handle formatting.

This is code is not about formatting the number, but it is just about validation.

The code checks that on the right input the user can't insert a number that is minor then the number that there is in the left input. About the naming of the function, what do you think about `checkMaxValueLimit`?

Reply:
Given that `withMaxValueLimit` is a boolean function, anything with `has_` or `is_` should work. Maybe `isOutsideBound`? Naming is hard so don't stress it.